websocket_service.go

```diff
@@ -62,6 +63,55 @@ func WsPartialDepthServe(symbol string, levels string, handler WsPartialDepthHan
 	return wsServe(cfg, wsHandler, errHandler)
 }
 
+func WsCombinedPartialDepthServe(symbols []string, levels []string, handler WsPartialDepthHandler, errHandler ErrHandler) (doneC, stopC chan struct{}, err error) {
```

Comment:
is it better to use a map for symbols and levels?
```
symbolLevels map[string]string
```

Reply:
yes, I think so. I will change